packages/OmegaPrint-Core.package/OPPrinter.class/instance/ObjectArrayLiteralElementNormal.with.and..st

```diff
@@ -0,0 +1,4 @@
+actions
+ObjectArrayLiteralElementNormal: aNode with: aStatement and: aLiteral
+
+	^ self value: aStatement
```

Comment:
Warum gibt es diese Methode? Ich hab da mal eine ähnliche gelöscht, weil wir gesagt haben, dass das eh passiert? Oder geht es hier darum, dass der Literal weggelassen wird?

Reply:
Jep genau, ansonsten bekommen wir da immer `statement .` zurück und Punkte wollen wir in dem Knoten darüber erst hinzufügen

Reply:
Ergaenzend zu @T4rikA's Aussage: Da wir nach dem letzten `ObjectArrayLiteralElement` keinen Punkt haben wollen, wir aber, wenn der Nutzer dort einen Punkt setzt, immer im Fall `--normal` von
```smalltalk
ObjectArrayLiteralElement =
	Statement "." --normal
	| Statement --finalExpression
```
sind, muessen wir zwingend alle Punkte hier entfernen, um sie dann oben mit einem schoenen `joinSeperatedBy:` ordentlich hinzuzufuegen.